apiserver/facades/schema.json

```diff
@@ -11544,17 +11544,6 @@
                         }
                     }
                 },
-                "CheckCredentialsModels": {
```

Comment:
Don't we need this method anymore?

Reply:
It's still on V6 facade for compatibility but removed from latest/V7 as it's not used. So old clients can still use it by asking for the V6 facade.